### ISSUE TITLE
Encrypt reminders

### ISSUE THREAD

**Issue by Aetheryx** [Aetheryx/remindme#16]
At the moment, reminders are stored as strings in the SQLite database. I would like to store these reminders encrypted.  
Since it's not exactly a self-contained method to have these reminders encrypted, I feel like this is too much of a vague issue to be done for the whole workflow of creating a reminder - which is why all I ask is `encrypt` and `decrypt` methods on the [main class](https://github.com/Aetheryx/remindme/blob/master/src/remindmebot.js). 
Something like:
```js
class RMB {
    constructor () {
        [...]
    }

    async onReady () {
        [...]
    }

    [.....]

    encrypt (string) {
        return [..encrypted..]
    }

    decrypt (encryptedString) {
        return [..decrypted string..]
    }
}
```

**Comment by Aetheryx** [comment]
goddamnit I was still typing

**Comment by Devoxin** [comment]
lmao

**Comment by dericcain** [comment]
@Aetheryx I can take this on.

**Comment by Aetheryx** [comment]
@dericcain Sounds good, have fun.

**Comment by dericcain** [comment]
@Aetheryx How do you want to handle the encryption key? There are a couple of ways to handle it:

1. Set that in a `.env` file and it will always be the same
2. Create a random key for each encryption and return that key, as well as the encrypted string, with the 'encrypt' method. You would need to store it in the DB and then use it for the `decrypt` method.

**Comment by Aetheryx** [comment]
@dericcain Would the latter have a noticeable performance hit, and is it a common method? I'm leaning towards the latter if it's something that wouldn't slow down too much, definitely feels alot safer than storing it in an unprotected file.

**Comment by dericcain** [comment]
@Aetheryx It should not be noticeable. You should be able to squeeze it into a DB call that you are already making so it is not an extra one and since you would be encrypting it anyways, you should be good to go.

**Comment by Aetheryx** [comment]
I see. Go for the latter then. 
The table is created [here](https://github.com/Aetheryx/remindme/blob/master/src/remindmebot.js#L65-#L70).  
I'm not sure how you would want to do the `decrypt` method when it's coming from the same call - I think having it take an object with the keys `encryptedReminder, encryptionKey` and having it return `decryptedReminder` should be fine.

**Comment by dericcain** [comment]
@Aetheryx I have got to travel home before I i do a pull, but what do you think of this?

https://github.com/dericcain/remindme/blob/master/src/remindmebot.js#L97

**Comment by Aetheryx** [comment]
Looks like something I'll be working with soon.

**Comment by Aetheryx** [comment]
Fixed in #17
